snippets/post_install_network_config

```diff
@@ -77,6 +77,9 @@ mv /etc/sysconfig/network.cobbler /etc/sysconfig/network
 # Also set the hostname now, some applications require it
 # (e.g.: if we're connecting to Puppet before a reboot).
 /bin/hostname $hostname
+        #if $osversion == "rhel7"
```

Comment:
Then have you tested:
hostnamectl --static set-hostname $hostname
This will have the same effect and add few verification step.
Or is there any issue to use hostnamectl in %post ?

Also I wonder where is set $hostname ? Because if it comes from the network, there is no point to hardcode a static hostname (in /etc/hostname). It makes sense only if it comes from a cobbler system entry.